src/components/color-pallete/color-palette.css

```diff
@@ -0,0 +1,61 @@
+/*color palette styling*/
+.color-palette {
+   width: 7rem;
+   height: max-content;
+   padding: 10px;
```

Comment:
Shouldn't padding change with font size? use rem.

Reply:
Thanks for pointing this out. Will work on it.